### PR TITLE
Fix warnings found in latest typescript compiler version.

### PR DESCRIPTION
**Pull request by rictic**
Split out of the commit in #300 since that's not gonna land today.

### CODE REVIEW

File: src/test/elements-metadata_tests.ts [hunk 1]
```diff
@@ -193,6 +193,6 @@ const paperButton: Elements = {
 // We want to keep these local values as is, because they serve as
 // documentation and compilation tests.
 // So there's this hack to prevent a warning about unused code.
-if (false !== true) {
+if (Math.random() > 1000) {
```
Comment (usergenic) [inline src/test/elements-metadata_tests.ts]: Shouldn't this be Math.random() < 1000? (if the same result expected as false !== true)

Comment (usergenic) [inline src/test/elements-metadata_tests.ts]: I recognize this probably doesn't matter.

**Comment by usergenic** [comment]
Apart from my pedantic nit about using Math.random() > 1000 for both true and false, this LGTM.

File: src/test/elements-metadata_tests.ts (hunk 1 above)
Comment (rictic) [inline src/test/elements-metadata_tests.ts]: Arg, the old code should have been `false === true`.

shamecube.gif

**Comment by rictic** [comment]
Thanks for the review!

**Pull request merged**